sql-plugin/src/main/scala/org/apache/spark/sql/rapids/datetimeExpressions.scala

```diff
@@ -374,19 +374,26 @@ abstract class UnixTimeExprMeta[A <: BinaryExpression with TimeZoneAwareExpressi
    rule: DataFromReplacementRule)
   extends BinaryExprMeta[A](expr, conf, parent, rule) {
 
-  var sparkFormat: String = _
-  var strfFormat: String = _
+  val optionalRightStringLit: Option[String] =
```

Comment:
Any reason for this not to be lazy, like the members below?

Reply:
Hmm. None that I can see. Good catch.

Reply:
Closing this in favour of a better solution.